protocol/avdl/chat1/gregor.avdl

```diff
@@ -78,6 +78,13 @@ protocol gregor {
         union { null, UnreadUpdate } unreadUpdate;
     }
 
+    record EphemeralPurgePayload {
+        @lint("ignore")
```

Comment:
@mmaxim should we also be sending down a deviceID? or will the `InboxVers` change be enough to force all devices to purge their cache?

Reply:
resolved offline -- added metadata of the last messageID that was purged to the `Conversation` record so offline devices can purge correctly